src/test/java/core/basesyntax/service/WriterServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.WriterServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceTest {
+    private static WriterService writerService;
+    private static final String REPORT = "banana,152" + System.lineSeparator() + "apple,90";
+    private static final String FILE_PATH = "src/test/resources/test_report.cvs";
```

Comment:
Also think about using path from classpath instead of hardcoding it.
```suggestion
    private static final String FILE_PATH = "src/test/resources/test_report.csv";
```